v3/plugins/redhat/rhamt/0.0.8/meta.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v2
+id: redhat.rhamt-vscode-extension
+version: 0.0.8
+type: VS Code extension
+name: rhamt
+displayName: Red Hat Application Migration Toolkit
+title: Red Hat Application Migration Toolkit
+description: Application Migration and Modernization using RHAMT
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+publisher: redhat
+repository: https://github.com/windup/rhamt-vscode-extension
+category: Other
+firstPublicationDate: "2019-06-19"
+spec:
+  endpoints:
+   - name: "configuration-endpoint"
+     public: true
+     targetPort: 61436
+     attributes:
+       protocol: http
```

Comment:
The CI is failing because the indentation is off here (lines 16-20 have an additional space, causing `yq` to choke on it).